src/telemetry/telemetry.c

```diff
@@ -72,6 +73,9 @@
 #define REQ_INSTANCE_METADATA "instance_metadata"
 #define REQ_TS_TELEMETRY_CLOUD "cloud"
 
+#define REQ_NUM_REPLICAS "num_replicas"
+#define REQ_IS_REPLICA "is_replica"
```

Comment:
A couple of issues:

* I think the term "replica" is actually very ambiguous. Is it a standby, async/sync standby, logical replica, etc? I suggest coming up with something more specific, like `num_standbys` if that's what we are looking for.
* Since previous refactorings of the telemetry tried to move away from the flat JSON, I think it could make sense to group this information in a sub-object. Otherwise, these fields might not end up next to each other and the JSON will eventually become unwieldy if we keep adding flat fields. The information will be harder to read and parse as related fields will not be  grouped together.